projects/storefrontapp-e2e-cypress/cypress/integration/regression/password-visibility/password-visibility.core-e2e-spec.ts

```diff
@@ -0,0 +1,37 @@
+import { viewportContext } from '../../../helpers/viewport-context';
+import { standardUser } from '../../../sample-data/shared-users';
+import { generateMail, randomString } from '../../../helpers/user';
+
+context('Password Visibility', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    beforeEach(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.clear();
+      });
+    });
+
+    it('should hide password by default', () => {
+      cy.visit('/login');
+      cy.get('button[aria-label="Show password"]').should('be.visible');
+    });
+
+    it('should show and hide password when toggled', () => {
+      cy.visit('/login');
+      cy.get('button[aria-label="Show password"]').click();
+
+      cy.get('button[aria-label="Hide password"]').click();
+    });
+
+    it('should verify on other pages', () => {
+      //on registration page
+      cy.visit('/login/register');
+      cy.get('button[aria-label="Show password"]').should('have.length', 2);
+
+      //on my account update password page
+      standardUser.registrationData.email = generateMail(randomString(), true);
+      cy.requireLoggedIn(standardUser);
```

Comment:
I believe you don't need to pass in anything 
you can just do `cy.requireLoggedIn`. 

Also, let's separate this test case.
For example, 1 for should hide password by default in register page and another one for should hide password by default in my account update password page.
Currently the test case just checks the default state of the password being hidden (which is fine) since we tested in the login page already

Reply:
Done!